lib/Core/Rpc/Message.php

```diff
@@ -4,6 +4,16 @@
 
 abstract class Message
 {
+    /**
+     * @var string
+     */
+    public $method;
+
+    /**
+     * @var array
+     */
+    public $params;
+
     /**
      * @var string
      */
```

Comment:
not sure if this is part of JSON RPC, but both Notification and Request have `method` and `params`, and this provides more polymorphism.